lib/mapprep.gi

```diff
@@ -1366,6 +1367,17 @@ InstallMethod( PrintObj,
     Print( "IdentityMapping( ", Source( id ), " )" );
     end );
 
+InstallMethod( String, 
+    "for identity mapping", 
+    [ IsGeneralMapping and IsOne ], 
+    # rank up, but just to exactly SUM_FLAGS, so that mappings in a special
+    # representation with a custom printing method still get that, even if
+    # the rank of IsGeneralMapping and IsOne happens to be increased a lot
+    {} -> SUM_FLAGS - RankFilter( IsGeneralMapping and IsOne ),
+  function ( id ) 
+    return StringFormatted( "IdentityMapping( {} )", Source(id) ); 
```

Comment:
I guess @ChrisJefferson and me disagree here what is best; personally, I would not use `StringFormatted` here, as fewer people are familiar with it, so I think the construction with `Concatenation` is more maintainable. However, the code is fine either way, and in the end it's a matter of taste. So, no need to change this again, I just wanted to get it off my chest (being one of these blunt germans without manners, please forgive me :-)